db/migrate/20170925164627_create_todos.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTodos < ActiveRecord::Migration[5.1]
+  def change
+    create_table :todos do |t|
+      t.string :title
+      t.text :description
+      t.timestamp :due_date
```

Comment:
`t.timestamp` causes your due dates to have a blank times:
![image](https://user-images.githubusercontent.com/17665837/30838226-3ab90fb6-a238-11e7-896f-f49c89aeb200.png)

`t.datetime` might be a better column data type.

Reply:
Hmm I was under the impression that datetime and timestamp were pretty much different names for the same thing (so that Rails can support different dbs). I'm not sure if it's the column data type that's the cause of a 'blank time', or just the fact that we're not letting the user input a time in our form yet.

Reply:
You're right. I looked into it and the only difference is how they are formatted and stored in the database.